### ISSUE TITLE
since project version -o semver gives wrong next version

### ISSUE THREAD

**Issue by davidolrik** [outofcoffee/since#1]
Given a repo with the following commits and tags (some commits obmitted):

- 0.0.1 First commit
- 0.1.0
- 0.1.1
- 0.1.2 Latest commit

I would assume that `since project version -o semver` would give me a next version of `0.1.3` but instead I get `0.1.0` as it sees `0.0.1` as the first commit.

```sh
$ since project version -o semver
DEBU[0000] most recent tag: 0.0.1
DEBU[0000] fetched 44 commits
DEBU[0000] commit types: [typo chore feat docs fix build]
DEBU[0000] bumping minor version
0.1.0
```

Using `commit-date` works - but shouldn't `semver` also use the latest commit as a starting point?

```sh
$ since project version -o commit-date
DEBU[0000] most recent tag: 0.1.2
DEBU[0000] fetched 15 commits
DEBU[0000] commit types: [docs fix build typo]
DEBU[0000] bumping patch version
0.1.3
```

**Comment by outofcoffee** [comment]
Hi @davidolrik - thank you for reporting this.

I can reproduce this with the following script:

```bash
#!/usr/bin/env bash
set -e

REPO_DIR="$( mktemp -d -t gittest )"
cd "${REPO_DIR}"
git init

touch README.txt
git add README.txt
git commit -a -m"First commit"
git tag 0.0.1

echo "first update" > README.txt
git commit -a -m"first update"
git tag 0.1.0

echo "second update" >> README.txt
git commit -a -m"second update"
git tag 0.1.1

echo "third update" >> README.txt
git commit -a -m"third update"
git tag 0.1.2

echo "Repo created at ${REPO_DIR}"
```

And then:

```bash
$  since project version -o semver
DEBU[0000] most recent tag: 0.0.1
...
```

This is definitely wrong 👀

**Comment by outofcoffee** [comment]
Fixed in [v0.15.0](https://github.com/release-tools/since/releases).

**Comment by davidolrik** [comment]
Works like a charm!
